webapp/src/containers/Rhs/projectList/index.tsx

```diff
@@ -88,27 +88,35 @@ const ProjectList = () => {
                 )
             }
             {
-                usePlugin.getApiState(plugin_constants.pluginApiServiceConfigs.getAllLinkedProjectsList.apiServiceName).isSuccess &&
-                data &&
-                (
-                    data?.length > 0 ?
-                        data?.map((item) => (
-                            <ProjectCard
-                                onProjectTitleClick={handleProjectTitleClick}
-                                projectDetails={item}
-                                key={item.projectID}
-                                handleUnlinkProject={handleUnlinkProject}
-                            />
-                        )) :
-                        (
-                            <EmptyState
-                                title='No Project Linked'
-                                subTitle={{text: 'You can link a project by clicking the below button or using the slash command', slashCommand: '/azuredevops link'}}
-                                buttonText='Link new project'
-                                buttonAction={handleOpenLinkProjectModal}
-                            />
-                        )
-                )
+                usePlugin.getApiState(plugin_constants.pluginApiServiceConfigs.getAllLinkedProjectsList.apiServiceName).isSuccess && (
```

Comment:
Same as above, better if we create a function that returns the state, we can reduce the length of this expression.